### PR TITLE
WIP: Add publishMesh example

### PR DESCRIPTION
**Pull request by nbbrooks**

### CODE REVIEW

**Comment by davetcoleman** [comment]
if you add a ``man.stl`` we can have a man bear pig mesh. just saying.

File: .travis.yml [hunk 1]
```diff
@@ -18,9 +18,13 @@ env:
...
     - TEST=clang-tidy-check
     - ROS_DISTRO=eloquent
+before_install:
+- git lfs install
+#- echo -e "machine github.com\n  login $GITHUB_TOKEN" > ~/.netrc
+- git lfs pull
 
 before_script:
-  - git clone -q -b ros2 --depth=1 https://github.com/ros-planning/moveit_ci.git .moveit_ci
+  - git clone -q -b ros2 --depth=1 https://github.com/nbbrooks/moveit_ci.git .moveit_ci
```
Comment (mlautman) [inline .travis.yml:27]: Please revert this. I'm assuming you have a PR over there as well?

File: src/rviz_visual_tools_demo.cpp [hunk 2]
```diff
@@ -355,7 +355,31 @@ class RvizVisualToolsDemo : public rclcpp::Node
...
+    const std::array<rvt::colors, 5> mesh_colors = { rvt::RED, rvt::ORANGE, rvt::YELLOW, rvt::GREEN,
+                                                     rvt::BLUE };
+    const std::array<double, 5> scales = { 0.025, 1.0, 1.0, 1.0, 1.0 };
+
+    for (size_t i = 0; i < meshes.size(); ++i)
+    {
+      visual_tools_->publishMesh(pose1 * Eigen::AngleAxisd(thetas[i], Eigen::Vector3d::UnitZ()),
+                                 "package://rviz_visual_tools/meshes/" + meshes[i] + ".stl",
+                                 mesh_colors[i], scales[i]);
+      if (i == 0.0)
```
Comment (mlautman) [inline src/rviz_visual_tools_demo.cpp:375]: Just do this before the for loop?

Comment (nbbrooks) [reply src/rviz_visual_tools_demo.cpp:375]: This is the pattern used everywhere else in the file, I'd say refactor it in a diff PR

File: .travis.yml (hunk 1 above)
Comment (nbbrooks) [reply .travis.yml:27]: I'm trying to debug several frustrating issues with moveit_ci right now which is why this is here

**Pull request closed without merging**